site/_includes/layouts/blog-post.njk

```diff
@@ -22,6 +22,13 @@
         fetchpriority="high"
       %}
     </div>
+
+  {% elseif heroYoutubeId %}
```

Comment:
I know it's already mixed in the codebase, sorry for being picky: as YAML and Nunjucks (by being a port of jinja2) are coming from Python it makes sense to use `snake_case` for variables used in this context.